File: app/src/main/java/ru/voronezhtsev/weatherapp/ru/voronezhtsev/weatherapp/db/ru/voronezhtsev/weatherapp/db/entities/Forecast.java
package ru.voronezhtsev.weatherapp.ru.voronezhtsev.weatherapp.db.ru.voronezhtsev.weatherapp.db.entities;

/**
 * Прогноз погоды, который лежит в БД
 *
 * @author Воронежцев Игорь on 16.12.2018.
 */
public class Forecast {
    private long mDateTime;
    private double mTemp;
    private double mTempMin;
    private double mTempMax;

    public long getDateTime() {
        return mDateTime;
    }

    public void setDateTime(long mDateTime) {
        this.mDateTime = mDateTime;
    }

    public double getTemp() {
        return mTemp;
    }

    public void setTemp(double mTemp) {
        this.mTemp = mTemp;
    }

    public double getTempMin() {
        return mTempMin;
    }

    public void setTempMin(double mTempMin) {
        this.mTempMin = mTempMin;
    }

    public double getTempMax() {
        return mTempMax;
    }

    public void setTempMax(double mTempMax) {
        this.mTempMax = mTempMax;
    }
}

File: app/src/main/java/ru/voronezhtsev/weatherapp/db/entities/Forecast.kt
package ru.voronezhtsev.weatherapp.db.entities

/**
 * Сущность прогноза погоды для хранения в БД
 *
 * @author Воронежцев Игорь on 12.01.19
 */
data class Forecast(
        /**
         * Время UNIX, UTC, мс
         */
        val dateTime: Long,
        /**
         * Температура в кельвинах
         */
        val temp: Double,
        /**
         * Минимальная температура, это отклонение от температуры, присущее большим городам
         * и мегаполисам
         */
        val tempMin: Double,
        /**
         * Максимальная температура, это отклонение от температуры, присущее большим городам
         * и мегаполисам
         */
        val tempMax: Double
)
